README.md

```diff
@@ -26,9 +26,9 @@ The Orbit Model contains four fundamental concepts: Gravity, Love, Reach and Orb
   - [Gravity](#gravity)
 - [Orbit levels](#orbit-levels)
   - [Orbit 4 - Observers](#orbit-4---observers)
-  - [Orbit 3 - User](#orbit-3---user)
-  - [Orbit 2 - Fans](#orbit-2---fans)
-  - [Orbit 1 - Ambassadors](#orbit-1---ambassadors)
+  - [Orbit 3 - Participants](#orbit-3---participants)
+  - [Orbit 2 - Contributors](#orbit-2---contributors)
+  - [Orbit 1 - Core](#orbit-1---core)
```

Comment:
Core might feel a little bit too strong for O1, as it’s in Open Source projects it’s usually a handful of people. E.g. for VueJS, the [core team](https://vuejs.org/v2/guide/team.html#active-core-team-members) is 20 members, but I’d gather that the Orbit 1 would be more than a hundred members big.

I don’t have a better suggestion though, as other words such as _supporters_, _advocates_, or _champions_ also have other meanings in some communities. So I’m OK staying with Core.